sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -629,14 +631,25 @@
      *   .map(Response::value);
      * </pre>
      *
-     * @return A {@link Flux} containing {@link KeyBase key} of all the keys in the vault.
+     * @return A {@link PagedFlux} containing {@link KeyBase key} of all the keys in the vault.
      */
-    public Flux<KeyBase> listKeys() {
+    public PagedFlux<KeyBase> listKeys() {
+        return new PagedFlux<>(() -> listKeysFirstPage(),
+            continuationToken -> listKeysNextPage(continuationToken));
```

Comment:
Just a formatting thing: I think it would be more readable to have both lambda statements on new lines.

Reply:
Move the `() ->` down to the next line too, so it is `() -> listKeysFirstPage(),`